contracts/ito.sol

```diff
@@ -453,43 +460,5 @@ contract HappyTokenPool is Initializable {
         // Gas Refund
         pool.packed1 = Packed1(DEFAULT_ADDRESS, 0);
         pool.packed2 = Packed2(0, 0);
-        for (uint256 i = 0; i < pool.exchange_addrs.length; i++) {
-            pool.exchange_addrs[i] = DEFAULT_ADDRESS;
-            pool.exchanged_tokens[i] = 0;
-            pool.ratios[i*2] = 0;
```

Comment:
No problem to clear `pool.ratios`, since Front-end can get `ratios` from the input params of `fill_pool()`

Reply:
updated